packages/tooltip/README.md

```diff
@@ -0,0 +1,3 @@
+# Tooltip Name
+
+Tooltip documentation
```

Comment:
Hold merge until I have time to add documentation and review them with @izharbuendia